packages/scratch-gui/webpack.config.js

```diff
@@ -99,6 +99,11 @@ const distConfig = baseConfig.clone()
             'scratch-gui': path.join(__dirname, 'src/index.ts')
         },
         output: {
+            // TODO: Test if this breaks the current version of -www!!
```

Comment:
@MarshTheTester, something to look out for once we merge this PR. If this is a problem, it'll likely break everything, but it's still worth looking out for subtle problems in the music extension or other non-obvious places.